src/main/java/com/shaft/tools/io/JSONFileManager.java

```diff
@@ -13,124 +13,141 @@
 import java.util.Map;
 
 public class JSONFileManager {
-    private final String jsonFilePath;
-    private FileReader reader = null;
+	private final String jsonFilePath;
+	private FileReader reader = null;
 
-    /**
-     * Creates a new instance of the test data json reader using the target json
-     * file path
-     *
-     * @param jsonFilePath target test data json file path
-     */
-    public JSONFileManager(String jsonFilePath) {
-        this.jsonFilePath = jsonFilePath;
-        try {
-            this.reader = new FileReader(jsonFilePath);
-        } catch (FileNotFoundException rootCauseException) {
-            ReportManager.log(rootCauseException);
-            ReportManager.log("Couldn't find the desired file. [" + jsonFilePath + "].");
-            Assert.fail("Couldn't find the desired file. [" + jsonFilePath + "].");
-        }
-        List<List<Object>> attachments = new ArrayList<>();
-        List<Object> testDataFileAttachment = null;
-        try {
-            testDataFileAttachment = Arrays.asList("Test Data", "JSON",
-                    new FileInputStream(jsonFilePath));
-        } catch (FileNotFoundException e) {
-            //unreachable code because if the file was not found then the reader would have failed at a previous step
-        }
-        attachments.add(testDataFileAttachment);
-        ReportManager.log("Successfully loaded the following test data file [" + jsonFilePath + "].", attachments);
-    }
+	/**
+	 * Creates a new instance of the test data json reader using the target json
+	 * file path
+	 *
+	 * @param jsonFilePath target test data json file path
+	 */
+	public JSONFileManager(String jsonFilePath) {
+		this.jsonFilePath = jsonFilePath;
+		try {
```

Comment:
let's move this entire try/catch block to a private method "initializeReader" so that we can reuse it below instead of duplicating the code.